build.gradle

```diff
@@ -41,7 +43,9 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.1.0'
     // These dependancies are for backwards compatibility with JUnit3/4
     testCompileOnly 'junit:junit:4.12'
-    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.1.0'}
+    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.1.0'
```

Comment:
Change is irrelevant to the user story. Maybe a bug report could be created to resolve.